### PR TITLE
Render when in "Visuals" tab of EID

### PR DESCRIPTION
**Pull request by Permamiss**
Update check so it renders while in "Visuals" tab of EID

### CODE REVIEW

**Comment by Permamiss** [comment]
Continuation of #123

File: main.lua [hunk 1]
```diff
@@ -529,10 +529,9 @@ local function onRender(t)
 	end
 	if EID.isHidden then
 		return
-	elseif EID.Config["HideInBattle"] then
```
Comment (wofsauge) [inline main.lua:532]: Why does it remove the code to hide it in battle?

**Review by wofsauge — CHANGES_REQUESTED** [review]

File: main.lua (hunk 1 above)
Comment (Permamiss) [reply main.lua]: > Why does it remove the code to hide it in battle?

I did not intend to do this. Honest mistake, sorry about this!

Comment (Permamiss) [reply main.lua]: Restored missing code HideInBattle code in current commit.

**Pull request merged**